base/uk.ac.stfc.isis.ibex.ui.motor/src/uk/ac/stfc/isis/ibex/ui/motor/views/MinimalMotorViewModel.java

```diff
@@ -117,11 +135,15 @@ private Color chooseColor() {
     }
 
     private String formatForMotorDisplay(String prefix, Double value) {
-        if (value != null) {
-            return String.format("%s: %.2f", prefix, value);
-        } else {
-            return "";
+        if (value == null) {
+        	return "";
         }
+        
+    	if (advancedMinimalMotorView) {
+            return String.format("%s %.3f", prefix, value);
```

Comment:
As far as I understood (from a discussion with you), 3 d.p. was a hard requirement for the reflectometers.

However...

No, there isn't always necessarily enough room. This is a fundamental issue with trying to put so much information into a small space. Finding the width of text before rendering is actually a really hard problem (for non fixed width fonts), so it's hard to tell how much of a number we would actually be able to fit in the space we have.

If the number is too long the last part of it will be cut off when it displays. This is the best compromise I was able to come up with, if you've got a better idea let's discuss...

Reply:
Display 3 d.p. on simple view as well as advanced